_submissions/round-12/1/2015-02-26-unix-shell-files-directories.md

```diff
@@ -3,11 +3,11 @@ date: 2015-02-26
 round: Round 12
 title: unix shell files directories
 author: Peter Fields
-permalink: /2015/02/unix-shell-files-directories/
+permalink: /2015/02/Unix-Shell-Files-Directories/
```

Comment:
all the other permalinks are all lower case - why change these to be title case?